src/world/area_kpa/kpa_81/A78510.c

```diff
@@ -4,9 +4,39 @@
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+extern s32 D_80240D98_A79258;
+extern s32 D_80240D9C_A7925C;
+
+// Needs data migrated
+#ifdef NON_MATCHING
+ApiStatus func_8024027C_A7873C(Evt* script, s32 isInitialCall) { 
+    s32* temp = script->ptrReadPos;
+    
+    if (isInitialCall != 0) {
```

Comment:
```suggestion
    if (isInitialCall) {
```
since we're just checking if a bool is true